src/SeleniumLibrary/keywords/element.py

```diff
@@ -731,39 +757,15 @@ def page_should_not_contain_image(self, locator, message='', loglevel='INFO'):
 
     @keyword
     def get_matching_xpath_count(self, xpath, return_str=True):
-        """Returns number of elements matching `xpath`
-
-        The default return type is `str` but it can changed to `int` by setting
-        the ``return_str`` argument to Python False.
-
-        One should not use the xpath= prefix for 'xpath'. XPath is assumed.
-
-        Correct:
-        | count = | Get Matching Xpath Count | //div[@id='sales-pop']
-        Incorrect:
-        | count = | Get Matching Xpath Count | xpath=//div[@id='sales-pop']
-
-        If you wish to assert the number of matching elements, use
-        `Xpath Should Match X Times`.
-        """
+        """Deprecated. Use `Get Matching Locator Count` instead."""
         count = len(self.find_element("xpath=" + xpath, first_only=False,
                                       required=False))
         return str(count) if is_truthy(return_str) else count
```

Comment:
Couldn't the implementation be changes to just `return self.get_matching_locator_count('xpath:' + xpath)`? Same with the other deprecated keyword below.

Reply:
Good idea, will do